### ISSUE TITLE
Cannot activate License on local instance

### ISSUE THREAD

**Issue by kateyork** [freescout-helpdesk/freescout#3367]
Hello!

I am having the issue that I can't activate Modules which I purchased.
I installed one DEV server and one PROD server. As the PROD Server has no access to the internet, I installed and activated the Modules on the DEV machine (which has internet access) and copied all modules from the /Modules folder to my PROD machine.
This was proposed in the Modules FAQ.
Unfortunately the Modules aren't activated then.

If I try to activate them, I get the following Error, which is actually expected.
"cURL error 28: Failed to connect to cdn.freescout.net port 443 after 10000 ms: Timeout was reached (see http://curl.haxx.se/libcurl/c/libcurl-errors.html)"

But how can I activate Modules on a Machine without any internet access?

PHP version: 8.2.10
FreeScout version: 1.8.95

**Comment by freescout-helpdesk** [comment]
https://freescout.net/modules-faq/#how-will-i-get-the-code-of-the-module-how-can-i-download-the-files-locally-as-a-backup

https://freescout.net/modules-faq/#download-in-local-network

**Comment by kateyork** [comment]
thank you for your fast answer.
But that's exactly what I did. I activated the Modules on my DEV instance and then copied the content of the /Modules folder to the PROD instance.
The Modules appear now on my PROD instance, but they are not activated. And trying to activate the modules doesn't work, because the PROD instance has no internet connection.

I hope you understand what I'm trying to say.

Thanks for your support!
